src/proof/tests/nova_tests_lem.rs

```diff
@@ -3246,7 +3248,14 @@ fn test_prove_head_with_sym_mimicking_value() {
 
 #[test]
 fn test_dumb_lang() {
-    use crate::{coprocessor::test::DumbCoprocessor, eval::tests::coproc::DumbCoproc};
+    use crate::coprocessor::test::DumbCoprocessor;
+
+    use crate::{self as lurk};
+
+    #[derive(Clone, Debug, Coproc)]
+    enum DumbCoproc<F: LurkField> {
+        DC(DumbCoprocessor<F>),
+    }
```

Comment:
Can we simply not define this and use `DumbCoprocessor` directly? It's being redefined in `src/lem/tests/eval_tests.rs`